### DISCUSSION TITLE
Can't mount Open Media Vault share on rAudio

### DISCUSSION THREAD

**Discussion by ywalgoch** [rern/rAudio#1446] [General]
Hi, these are the CIFS share details on OMV:

![image](https://github.com/rern/rAudio/assets/147339868/4bad3efe-6864-4c6d-a812-e4eb4f2a9ac4)

No matter what I put in here:

![image](https://github.com/rern/rAudio/assets/147339868/5403007f-c8b1-474c-b70a-881ff154f6f8)

I just get 'Mount failed: no such file or directory'

Can anyone help please, or at least point me in the direction of some documentation? Many thanks.

**Comment by ceejayemm** [comment]
Hi

I use OMV (v6.x) CIFS shares with rAudio and have no problems.  You need:

Name:  any name you wish but you MUST supply a name as this is mandatory
Server IP: provide the IP number of your OMV instance. Mine is 192.168.86.250, yours might be different
Share: The name of music library share which in your case appears to be 'hippo'
User:  In OMV make sure that you have added a user to the CIFS share and use this username here.  Make sure the user has at least read, and preferably read/write, access to the share.
Password: the password for the OMV user above
Options: add any OMV option you may have used in OMV, this includes the CIFS/Samba version you are supporting from OMV. In my case I use 'vers=3.1.1' as this is the CIFS/Samba level I use in OMV

If you still have problems post a screenshot of the completed CIFS dialogue panel as its very difficult to be more specific if we can't see the actual options you are actually using.

Chris

**Comment by ywalgoch** [comment]
" Make sure the user has at least read, and preferably read/write, access to the share."

This I did, only to have OMV refuse to apply the configuration change for some reason I can't ascertain. I've joined the OMV forum and have enquired over there as to how to fix this... will report back once this is resolved!
